### PR TITLE
add zh translation content/zh/docs/reference/glossary/cri-o.md

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/glossary/cri-o.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!-- A tool that lets you use OCI container runtimes with Kubernetes CRI. -->
+该工具可让您通过 Kubernetes CRI 使用 OCI 容器运行环境。
+
+<!--more-->
+
+<!-- CRI-O is an implementation of the {{< glossary_tooltip term_id="cri" >}}
+to enable using {{< glossary_tooltip text="container" term_id="container" >}}
+runtimes that are compatible with the Open Container Initiative (OCI)
+[runtime spec](http://www.github.com/opencontainers/runtime-spec). -->
+CRI-O 是 {{< glossary_tooltip term_id="cri" >}} 的实现，可启用与开放容器倡议 Open Container Initiative (OCI) 兼容的 {{< glossary_tooltip text="container" term_id="container" >}} 运行环境 [运行时规范](http://www.github.com/opencontainers/runtime-spec)。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/cri-o.md]:
```suggestion
CRI-O 是 {{< glossary_tooltip term_id="cri" >}} 的实现，可启用与开放容器倡议 Open Container Initiative（OCI）兼容的 {{< glossary_tooltip text="container" term_id="container" >}} 运行环境[运行时规范](http://www.github.com/opencontainers/runtime-spec)。
```

File: content/zh/docs/reference/glossary/cri-o.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<!-- Deploying CRI-O allows Kubernetes to use any OCI-compliant runtime as the container
+runtime for running {{< glossary_tooltip text="Pods" term_id="pod" >}}, and to fetch
+OCI container images from remote registries. -->
+部署 CRI-O 允许 Kubernetes 使用任何符合 OCI 运行环境，作为容器运行环境去运行 {{< glossary_tooltip text="Pods" term_id="pod" >}}，并从远程注册表获取 OCI 容器映像。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/cri-o.md]:
```suggestion
部署 CRI-O 允许 Kubernetes 使用任何符合 OCI 运行环境，作为容器运行环境去运行 {{< glossary_tooltip text="Pods" term_id="pod" >}}，并从远程注册表获取 OCI 容器镜像。
```

**Comment by xichengliudui** [comment]
@tanjunchen ,need update.

**Comment by tanjunchen** [comment]
@xichengliudui  Thanks

**Pull request merged**